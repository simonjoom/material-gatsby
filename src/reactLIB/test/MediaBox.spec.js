import React from 'react';
import { shallow } from 'enzyme';
import MediaBox from '../src/MediaBox';
import mocker from './helper/mocker';

describe('<MediaBox />', () => {
  let wrapper;
  const materialboxMock = jest.fn();
  const restore = mocker('materialbox', materialboxMock);

  afterAll(() => {
    restore();
  });

  test('renders', () => {
    wrapper = shallow(
      <MediaBox
        className="more"
        src="image.jpg"
        caption="A demo media box1"
        width="650"
      />
    );

    expect(wrapper).toMatchSnapshot();
  });
});
