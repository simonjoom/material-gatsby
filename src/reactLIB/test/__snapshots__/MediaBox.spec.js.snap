// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MediaBox /> renders 1`] = `
<img
  className="materialboxed more"
  data-caption="A demo media box1"
  src="image.jpg"
  width="650"
/>
`;
